src/sentry/workflow_engine/models/alertrule_detector.py

```diff
@@ -0,0 +1,32 @@
+from django.db.models import CheckConstraint, Q
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import DefaultFieldsModel, FlexibleForeignKey, region_silo_model
+
+
+@region_silo_model
+class AlertRuleDetector(DefaultFieldsModel):
+    """
+    A lookup model for rules and detectors.
+    """
+
+    __relocation_scope__ = RelocationScope.Excluded
```

Comment:
Will this change to org scope in the future? Having alert rules lose their conditions on import/export is going to be unpleasant for self-hosters moving to saas.

Reply:
updated 👍